### ISSUE TITLE
PAM get_topic_words cannot extract word distribution for sub-topics

### ISSUE THREAD

**Issue by by1228** [bab2min/tomotopy#2]
@bab2min The get_topic_words and get_topic_word_dist methods in PAM (inherited from LDA) only only user to extract the word-distribution for super-topics but not the sub-topics.

**Comment by bab2min** [comment]
Thank you for reporting the issue.
I've discovered the following problem in version 0.1.3 and below.

```
from tomotopy import *
mdl = PAModel(k1=5, k2=25) # k1: the number of super-topics, k2: the number of sub-topics
# ...
# mdl.add_doc(...)
# ...
mdl.train(iter=1000)
for k2 in range(25):
    print('Subtopic', k2, ':', mdl.get_topic_words(k2))
    # mdl.get_topic_words() raises an exception when k2 >= 5
```
get_topic_words() in PAModel should return top words for each sub-topic, but it raises an exception when topic number >= k1.

This issues will be fixed in the next update. Thank you again for discovering the bug.

**Comment by bab2min** [comment]
The issue was fixed at 0.1.4.
